mizar/networkpolicy/networkpolicy_util.py

```diff
@@ -30,26 +30,76 @@
 
 
 class NetworkPolicyUtil:
-    def handle_networkpolicy_create_update(self, name, pod_label_dict, policy_types):
+    def update_networkpolicy_endpoints_mapping_and_return_affected_endpoint_names(self, policy_name, pod_label_dict):
```

Comment:
this func name seems too long. From the name, it seems to achieve two things - update the mapping, and get affected endpoints. If so, please consider split to 2 functions; otherwise, please consider renaming it to reflect its purpose.

Reply:
Fixed